.gitignore

```diff
@@ -0,0 +1,12 @@
+*.txt
+*.csv
+*.pdf
+*.ipynb
+*.dict
+*.mm
+*.index
+*.pkl
+.ipynb_checkpoints
+__pycache__/
+.idea/
+*.ini
```

Comment:
config file should be added to the remote repo!

Reply:
updated now